101-webappazure-oms-monitoring/azuredeploy.json

```diff
@@ -11,7 +11,7 @@
 		},
 		"workspaceLocation": {
 			"type": "string",
-			"defaultValue": "",
+			"defaultValue": "[resourceGroup().location]",
```

Comment:
Microsoft.OperationalInsights/workspaces is not globally available so you need to use allowedValues (for both clouds)